eyieko/landville-frontend
# #166949157 Enable users to accept terms of use on registration

## Description ##
- Update the current user registration functionality to enforce acceptance of terms and conditions.

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##


## How to test: ##
- Clone the repo
- Activate your virtual environment 
- Install dependencies using `npm install`
- Checkout branch ft-accept-terms-conditions-166949157
- Navigate to the registration page and you will see a link to the `terms of conditions`
-  Check the checkbox to accept the terms of registration before registration.

## Related Pivotal Tracker stories ##
[#166949157](https://www.pivotaltracker.com/story/show/166949157)
